lib/test.gi

```diff
@@ -22,9 +22,10 @@ InstallGlobalFunction(ParseTestInput, function(str, ignorecomments)
   while i <= Length(lines) do
     if i = 1 and  Length(lines[1]) > 0 and lines[1][1] = '#' then
       if ignorecomments = true then
-        # ignore comment lines at beginning of file
-        while i <= Length(lines) and Length(lines[i]) > 0 and
-              lines[i][1] = '#' do
+        # ignore comment lines and empty lines at beginning of file
+        while i <= Length(lines) and
+            ((Length(lines[i]) > 0 and lines[i][1] = '#') or lines[i] = "")
```

Comment:
Simpler expresdion: `Length(lines[i]) = 0 or lines[i][1] = '#'`

Reply:
Fixed (also the failing test)